Configuration/Eras/python/Era_Run2_2016_cff.py

```diff
@@ -8,7 +8,8 @@
 from Configuration.Eras.Modifier_run2_muon_2016_cff import run2_muon_2016
 from Configuration.Eras.Modifier_run2_egamma_2016_cff import run2_egamma_2016
 from Configuration.Eras.Modifier_run2_L1prefiring_cff import run2_L1prefiring
+from Configuration.Eras.Modifier_pixel_2016_cff import pixel_2016
 
 Run2_2016 = cms.ModifierChain(run2_common, run2_25ns_specific,
- stage2L1Trigger, ctpps_2016, run2_HLTconditions_2016, run2_muon_2016, run2_egamma_2016, run2_L1prefiring)
+ stage2L1Trigger, ctpps_2016, run2_HLTconditions_2016, run2_muon_2016, run2_egamma_2016, run2_L1prefiring, pixel_2016)
```

Comment:
I suppose `pixel_2016` should be removed from `Run2_2017`?
https://github.com/cms-sw/cmssw/blob/72e9ddf93ef02082d4c7abc4569fb3b43ea2ea7b/Configuration/Eras/python/Era_Run2_2017_cff.py#L21

(on the other hand, with this many year-specific modifiers maybe we should consider reorganizing the era definitions by separating the modifiers that carry on to the next year and the modifiers that are specific for that year; clearly not for this PR)

Reply:
> I suppose pixel_2016 should be removed from Run2_2017?

why?
anyway it is already overwritten by:
https://github.com/cms-sw/cmssw/blob/72e9ddf93ef02082d4c7abc4569fb3b43ea2ea7b/SimGeneral/MixingModule/python/SiPixelSimParameters_cfi.py#L112

Reply:
Easier maintenance. If the `pixel_2016` gets used more, one doesn't have to undo the modifications with `phase1Pixel`.

In general my understanding for the modifiers with the year postfix is that they are specific for that year and are not intended to be enabled for subsequent years (but I could be wrong, but then I'd say we have a mess).

Reply:
OK, thanks. I was actually asking myself if undoing it was the recommend way when preparing the PR, but then decided for economy not to do it. If you confirm that's the desired outcome I will update the branch. I need also updated GTs from @christopheralanwest. I will push all the updates at the same time.

Reply:
comment has been taken care of by https://github.com/cms-sw/cmssw/pull/27600/commits/1bc910f8fee7055135bcd89f32211c81699b4a7f